internal/app/app.go

```diff
@@ -3,6 +3,15 @@ package app
 import (
 	"context"
 	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/avast/retry-go/v4"
+
+	"github.com/neutron-org/neutron-query-relayer/internal/subscriber/querier/client/query"
+
+	rpcclienthttp "github.com/tendermint/tendermint/rpc/client/http"
```

Comment:
please run `goimports -w -local github.com/neutron-org .`

Reply:
I've already run it and it changed nothing(